src/SarifWeb/Startup.cs

```diff
@@ -36,8 +37,18 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
                 app.UseHsts();
             }
+
             app.UseHttpsRedirection();
-            app.UseStaticFiles();
+
+            app.UseStaticFiles(new StaticFileOptions
+            {
+                OnPrepareResponse = ctx =>
+                {
+                    const int durationInSeconds = 60 * 60 * 24 * 365;
```

Comment:
>60 * 60 * 24 * 365 [](http://example.com/codeflow?start=50&length=18)

do we really want such long cache duration?
if some static files get updated without changing names in server side, client side will not get the latest files unless after 1 year, or force refreshing the browser?

Reply:
I also think it is too long, shorten to 30 days.